core/domain/collection_domain.py

```diff
@@ -342,7 +470,15 @@ def to_yaml(self):
 
     @classmethod
     def _convert_v1_dict_to_v2_dict(cls, collection_dict):
-        """Converts a v1 collection dict into a v2 collection dict."""
+        """Converts a v1 collection dict into a v2 collection dict.
+
+        Args:
+            collection_dict: dict. The dictionary representation of collection.
```

Comment:
The dict representation of a collection with schema version v1.

(DItto below -- two places.)